GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.js

```diff
@@ -271,8 +321,22 @@ gdjs.WebsocketDebuggerClient.prototype.sendProfilerOutput = function(
   );
 };
 
-// This is an alternative to JSON.stringify that ensure that circular reference
-// are replaced by a placeholder.
+/**
+ * A Function used to replace Cycles with a default name.
+ * @callback cycleReplacer
```

Comment:
Name types in PascalCase.
I would also prefix it by `DebuggerClient`, because types are globals (at least for now, will be easy to replace in the future).